ocdaction/ocdaction/templates/core/contact.html

```diff
@@ -5,6 +5,23 @@
 {% block main %} 
     <section>
  		<h1>Contact</h1>
-    </section> 
+        <div class="col-xs-offset-2 col-xs-8 col-sm-offset-3 col-sm-6 col-md-offset-4 col-md-4 col-lg-offset-4 col-lg-4">
+        <form role="form" method="post">
+         {% csrf_token %}
```

Comment:
Please remove the crf token and form as p as this page is not a django form, there is no form associated with this page in the backend